components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -0,0 +1,288 @@
+            this.setState({emailError: Utils.localizeMessage('next_steps_view.invite_members_step.tooManyEmails', 'Invitations are limited to 10 email addresses.')});
+            return;
+        }
+
+        if (value.indexOf(' ') !== -1 || value.indexOf(',') !== -1) {
+            const emails = value.split(/[\s,]+/).filter((email) => email.length).map((email) => ({label: email, value: email, error: !isEmail(email)}));
+            this.setState({emails: [...this.state.emails, ...emails], emailInput: '', emailError: undefined});
+        } else {
+            this.setState({emailInput: value, emailError: undefined});
+        }
+    }
+
+    onChange = (value: SelectionType[], action: ActionMeta<SelectionType[]>) => {
+        if (action.action !== 'remove-value' && action.action !== 'pop-value') {
+            return;
+        }
+
+        if (!value.some((email) => email.error)) {
+            this.setState({emailError: undefined});
+        }
+
+        this.setState({emails: value});
+    }
+
+    onBlur = () => {
+        if (this.state.emailInput) {
+            const emails = this.state.emailInput.split(/[\s,]+/).filter((email) => email.length).map((email) => ({label: email, value: email, error: !isEmail(email)}));
+            this.setState({emails: [...this.state.emails, ...emails], emailInput: '', emailError: undefined});
+        }
+    }
+
+    sendEmailInvites = async () => {
+        if (this.state.emails.some((email) => email.error)) {
+            this.setState({emailError: Utils.localizeMessage('next_steps_view.invite_members_step.invalidEmail', 'One or more email addresses are invalid'), emailsSent: undefined});
+            return;
+        }
+
+        const emails = this.state.emails.map((value) => value.value);
+        const {data, error} = await this.props.actions.sendEmailInvitesToTeamGracefully(this.props.team.id, emails);
+
+        if (error || !data.length || data.some((result) => result.error)) {
+            this.setState({emailError: Utils.localizeMessage('next_steps_view.invite_members_step.errorSendingEmails', 'There was a problem sending your invitations. Please try again.'), emailsSent: undefined});
+            return;
+        }
+
+        this.setState({emailError: undefined, emailsSent: data.length}, () => {
+            setTimeout(() => this.setState({emailsSent: undefined}), 4000);
+        });
+    }
+
+    onSkip = () => {
+        this.props.onSkip(this.props.id);
+    }
+
+    onFinish = () => {
+        this.props.onFinish(this.props.id);
+    }
+
+    copyLink = () => {
+        const textField = document.createElement('textarea');
+        textField.innerText = this.getInviteURL();
+        textField.style.position = 'fixed';
+        textField.style.opacity = '0';
+
+        document.body.appendChild(textField);
+        textField.select();
+
+        try {
+            this.setState({copiedLink: document.execCommand('copy')});
+        } catch (err) {
```

Comment:
Do we need to provide user feedback if there is an error?

Reply:
I think we're okay here, there is user feedback when the link is successfully copied, which is how most sites seem to do it.